.storybook/pages/FeedbackOverview/FeedbackOverview.tsx

```diff
@@ -38,1679 +37,535 @@ export interface Props {
-                            margin: 0,
-                            flexShrink: 0,
-                          }}
-                        >
-                          &nbsp;
-                        </div>
-                        <div
-                          className="fpo"
-                          style={{
-                            height: '1.25rem',
-                            width: '1.25rem',
-                            padding: 0,
-                            margin: 0,
-                            flexShrink: 0,
-                          }}
-                        >
-                          &nbsp;
-                        </div>
-                        <div
-                          className="fpo"
-                          style={{
-                            height: '1.25rem',
-                            width: '1.25rem',
-                            padding: 0,
-                            margin: 0,
-                            flexShrink: 0,
-                          }}
-                        >
-                          &nbsp;
-                        </div>
-                      </div>
-                    </TableCell>
-                    <TableCell data-heading="Final product">
-                      {' '}
-                      <Icon
-                        color={EdsThemeColorIconNeutralSubtle}
-                        name="star-outline"
-                        purpose="decorative"
-                        size="1.5rem"
-                      />{' '}
-                      <Icon
-                        color={EdsThemeColorIconNeutralSubtle}
-                        name="star-outline"
-                        purpose="decorative"
-                        size="1.5rem"
-                      />{' '}
+      if (window.innerWidth < tableBreakpoint && isTable) {
+        setIsTable(false);
+      }
+    },
+    200,
+    { leading: true },
+  );
+  useEffect(() => {
+    updateScreenSize();
+    window.addEventListener('resize', updateScreenSize);
+    return () => {
+      window.removeEventListener('resize', updateScreenSize);
+    };
+  }, [updateScreenSize]);
+
+  const studentCheckpoints = [
+    {
+      student: 'Araya, Raquel',
+      checkpoints: 10,
+    },
+    {
+      student: 'Jesse Banet',
+      checkpoints: 5,
+    },
+    {
+      student: 'Julie Davis',
+      checkpoints: 10,
+    },
+    {
+      student: 'Hussain, Adnan',
+      checkpoints: 10,
+    },
+    {
+      student: 'Ilango, Megha',
+      checkpoints: 10,
+    },
+    {
+      student: 'Jaffer, Arman',
+      checkpoints: 10,
+    },
+    {
+      student: 'Kang, Michelle',
+      checkpoints: 10,
+    },
+    {
+      student: 'Lewine, Chris',
+      checkpoints: 10,
+    },
+    {
+      student: 'Luo, Celia',
+      checkpoints: 10,
+    },
+  ];
+
+  const feedbackOverviews = studentCheckpoints.map(
+    ({ student, checkpoints }) => {
+      const status = Array(checkpoints).fill(
```

Comment:
Instead of a bunch of elements, used this as a condensed way to replicate all the fpo blocks, saving hundreds of lines of the same element code